### ISSUE TITLE
Notification for notes on tasks you're assigned to in Hubble

### ISSUE THREAD

**Issue by lamepennies** [juno-fx/report#61]
**Is your feature request related to a problem? Please describe.**
Just to speed up looking for things

**Describe the solution you'd like**
Perhaps an inbox on the hubble page with the notes you're tagged on so you don't need to look through everything.

**Describe alternatives you've considered**

**Additional context**

**Comment by aldmbmtl** [comment]
This is actively being worked on here https://github.com/juno-fx/report/issues/55. I need to think of a proper way to do this so it is easier to find your notes. Got any suggestions? 

These are my ideas
* When in your workstation view, I can put a little note icon next to the hubble button that says "Notes". It can then slide out from the right a little panel that has a timeline of the notes that you have and on which shots.
* I can add a notes view to the top of the production window. So next to Structure and Versions. There you would get a "timeline" of notes coming in so you can see what notes came in and if you have seen them. 
* We leave the notes where they are in Hubble and just add a notes view in File Browser. File Browser is getting quite cluttered though so maybe we would need a new widget all together.
* We do all of the above?
* Do something else entirely?

**Comment by lamepennies** [comment]
I was thinking just a notes tab in the Hubble sidebar. Couldn't you just add the date and time info in that? I figure whatever is cleaner

I think notes in the file browser could be good too.

**Comment by aldmbmtl** [comment]
I could do that. I will need to check our note object. I am pretty sure we do have the publish time on it so I could just have it on a sidebar.

**Comment by skeleturtle** [comment]
last comment was from way back in june. Is this still needed? If so are there any roadblocks or things needed from pipeline?

**Comment by lamepennies** [comment]
Yeah. Right now everyone gets notifications. It should go to the artist and maybe a prod person or group?

**Comment by skeleturtle** [comment]
- Need to be able to clear notifications
- Need adjust who gets notified.
- Sorting/filters needs to be reviewed and set up between internal/external 

I will need to touch base with @aldmbmtl to learn more about how this is set up so I can help out.

**Comment by skeleturtle** [comment]
Once rocket chat is set up, we can also potentially look at providing notifications to the users via chat
